bench/db_bench.ml

```diff
@@ -400,8 +419,13 @@ let input =
   in
   Arg.(value & opt options `Minimal & info [ "b"; "bench" ] ~doc)
 
+let json_output =
+  let doc = "Filename where json output should be written" in
+  Arg.(value & opt string "" & info [ "j"; "json" ] ~doc)
```

Comment:
```suggestion
  Arg.(value & opt (some non_dir_file) None & info [ "j"; "json" ] ~doc)
```
I think such null values should be exposed in the type. You could make `json_file` a `string option` with something like this. Also, Cmdliner has a `non_dir_file` Arg that will do a bit of validation for you (and improve the `--help` docs).

Reply:
as @CraigFe suggested :)